source/dotnet/Library/AdaptiveCards.Rendering.Wpf/AdaptiveRenderContext.cs

```diff
@@ -120,7 +122,16 @@ public virtual dynamic MergeInputData(dynamic data)
         /// </summary>
         public FrameworkElement Render(AdaptiveTypedElement element)
         {
-            return ElementRenderers.Get(element.GetType())(element, this);
+            var renderer = ElementRenderers.Get(element.GetType());
+            if (renderer != null)
+            {
+                return renderer.Invoke(element, this);
+            }
+            else
+            {
+                Warnings.Add(new AdaptiveViolation(-1, $"No renderer for element type '{element.Type}'"));
```

Comment:
The integer Code should be passed in via a `readonly` property, otherwise a value of -1, and later 5 are meaningless and prone to mistakes/conflicts

Reply:
Yea those are temp, we will standardize these. I'll add TODOs